.github/CODEOWNERS

```
@@ -1 +1 @@
-* @exercism/maintainers-admin
+* @verdammelt @TheLostLambda @exercism/maintainers-admin
```

I'd rather it was this. This means we don't get notified about stuff.

```suggestion
* @verdammelt @TheLostLambda 
.github/CODEOWNERS @exercism/maintainers-admin
```